src/OversetSimulation.h

```diff
@@ -99,6 +103,12 @@ class OversetSimulation
         m_nw_start_rank = start_ranks;
         m_num_nw_solvers = m_nw_start_rank.size();
     }
+
+    void set_holemap_alg(int alg)
```

Comment:
here you are taking an int, but it comes in as a bool, but you use it as an int.

Reply:
Alright. I think there is automatic casting, but I should probably just explicitly cast the boolean to an int inside `set_holemap_alg`.

Reply:
Or just keep it as an int from the start

Reply:
With the latest changes, the yaml file reads in a boolean, and an integer of 1 is passed to `set_holemap_alg` if the boolean is true.